packages/editor-ui/src/views/SettingsUsageAndPlan.vue

```diff
@@ -1,58 +1,61 @@
 <script lang="ts" setup>
-import { computed, onMounted, ref, watch } from 'vue';
+import { computed, onMounted, ref } from 'vue';
 import { useRoute, useRouter } from 'vue-router/composables';
-import { Notification } from "element-ui";
+import { Notification } from 'element-ui';
 import { UsageTelemetry, useUsageStore } from '@/stores/usage';
-import { UsageState } from "@/Interface";
-import { telemetry } from "@/plugins/telemetry";
+import { telemetry } from '@/plugins/telemetry';
 
 const usageStore = useUsageStore();
 const route = useRoute();
 const router = useRouter();
 
-const queryParamCallback = ref<string>(`callback=${encodeURIComponent(`${window.location.origin}${window.location.pathname}`)}`);
+const queryParamCallback = ref<string>(
+	`callback=${encodeURIComponent(`${window.location.origin}${window.location.pathname}`)}`,
+);
 const viewPlansUrl = computed(() => `${usageStore.viewPlansUrl}&${queryParamCallback.value}`);
 const managePlanUrl = computed(() => `${usageStore.managePlanUrl}&${queryParamCallback.value}`);
 const activationKeyModal = ref(false);
 const activationKey = ref('');
 const activationKeyInput = ref<HTMLInputElement | null>(null);
 
-const onLicenseActivation = () => {
-	usageStore.activateLicense(activationKey.value).then(() => {
-		activationKeyModal.value = false;
-	});
-};
-
-onMounted(async () => {
-	usageStore.setLoading(true);
-	if(route.query.key) {
-		await usageStore.activateLicense(route.query.key as string).then(() => router.replace({ query: {} }));
-	} else if(usageStore.canUserActivateLicense) {
-		await usageStore.refreshLicenseManagementToken();
-	} else {
-		await usageStore.getLicenseInfo();
-	}
-	usageStore.setLoading(false);
-});
-
-watch(() => usageStore.error, (error: UsageState['error']) => {
-	if(error?.message) {
+const showNotification = () => {
+	if (usageStore.error) {
 		Notification.error({
-			title: 'Error',
-			message: error.message,
+			title: usageStore.error.name,
+			message: usageStore.error.message,
 			position: 'bottom-right',
 		});
-	}
-});
-
-watch(() => usageStore.success, (success: UsageState['success']) => {
-	if(success?.title && success?.message) {
+	} else if (usageStore.success) {
 		Notification.success({
-			title: success.title,
-			message: success.message,
+			title: usageStore.success.title,
+			message: usageStore.success.message,
 			position: 'bottom-right',
 		});
 	}
+};
+
+const onLicenseActivation = async () => {
+	try {
+		await usageStore.activateLicense(activationKey.value);
+		activationKeyModal.value = false;
+	} catch (e) {}
+	showNotification();
+};
```

Comment:
not really what we discussed... it was better before.. i thought we would remove success and error completely from store.. 

```
const onLicenseActivation = async () => {
	try {
		await usageStore.activateLicense(activationKey.value);
		activationKeyModal.value = false;
                 Notification.success({..})
	} catch (e) {
                 Notification.error({...})
        }
};
```